app/src/main/kotlin/com/wire/android/ui/home/conversations/details/GroupConversationDetailsViewModel.kt

```diff
@@ -165,10 +166,10 @@ class GroupConversationDetailsViewModel @Inject constructor(
                         protocolInfo = groupDetails.conversation.protocol,
                         areAccessOptionsAvailable = groupDetails.conversation.isTeamGroup(),
                         isGuestAllowed = groupDetails.conversation.isGuestAllowed() || groupDetails.conversation.isNonTeamMemberAllowed(),
-                        isServicesAllowed = groupDetails.conversation.isServicesAllowed(),
+                        isServicesAllowed = groupDetails.conversation.isServicesAllowed() && !isMLSTeam && !isMLSConversation,
                         isUpdatingNameAllowed = isSelfAnAdmin && !isSelfExternalMember,
                         isUpdatingGuestAllowed = isSelfAnAdmin && isSelfInOwnerTeam,
-                        isUpdatingServicesAllowed = isSelfAnAdmin,
+                        isUpdatingServicesAllowed = isSelfAnAdmin && !isMLSTeam && !isMLSConversation,
```

Comment:
Question: Does this mean that teams, with both protocols available, won't be able to add bots (or edit conversations with them) for old/non-migrated Proteus conversations ? If that's the case just checkin if this is what we really want

Reply:
basically yes, if the default protocol of the team is MLS (even if the conversation is Proteus yet) we disable services, that's what I've been told :)